### DISCUSSION TITLE
[2.0] Site-wide configuration?

### DISCUSSION THREAD

**Discussion by sleiner** [pdm-project/pdm#1199] [Ideas]
In PDM 2.0, there are two "layers" of configuration as far as I see (ranked by descending priority):
 - Project Configuration (which is, of course, project-specific)
 - Home Configuration (which is user specific)

I'm proposing to add a third layer, located at `platformdirs.site_config_dir("pdm")` which serves as the lowest-priority layer.

## Why would this be helpful?

For multi-user setups, it would make sense for the system administrator to set some default settings, e.g. the default Python path or a potential PyPI mirror.

What do you think of this?

**Comment by frostming** [comment]
SGTM, but it will be read-only from CLI

**Reply by sleiner** [reply]
Read-only from CLI seems sensible 👍🏻

**Reply by sleiner** [reply]
Should I convert this to an issue?

**Reply by frostming** [reply]
Sure
